<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QDialog" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="seeProductsButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortProductButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>SortMyProducts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchProductButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addProductButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add_Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteProductButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete_Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setProductQuantityPriceButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Employiee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="totalPaymentsButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Total worth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkoutButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Check out Product</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
